setup.py

```diff
@@ -3,9 +3,9 @@
 
 
 def get_target_version():
-    build_nr = os.environ.get("GITHUB_RUN_NUMBER", "0+dev")
-    version = "0.1"
-    return "{}.{}".format(version, build_nr)
+    pyvespa_version = os.environ.get("PYVESPA_VERSION", "0+dev")
+    build_nr = os.environ.get("TRAVIS_BUILD_NUMBER", "0+dev")
+    return "{}.{}".format(pyvespa_version, build_nr)
```

Comment:
I don't think `0+dev.0+dev` is a valid version format, so I think you need a different default for `pyvespa_version`.